packages/spanner/system-test/spanner.js

```diff
@@ -1034,9 +1035,56 @@ var spanner = new Spanner(env);
 
             transaction.commit(function(err) {
               assert.ifError(err);
+              firstTransaction.commit(assert.ifError);
```

Comment:
> I dont understand why do you need to call commit here?

This could be a misunderstanding by me - I was working with the notion that I needed to wrap up a second transaction before committing the first in order to get an aborted error? I was just passing the first transaction. Since everything is async, I need to pass some sort of callback in order to ensure the commit order is correct.